web_app/dist/title.css

```diff
@@ -0,0 +1,47 @@
+html {
+    
+}
+
+body {
+    background: #93ca68;
+    color: white;
+}
+
+.title-page-font {
+    font-family: "Raleway", Helvetica, Arial, sans-serif;
+}
+
+.title-page-spacing {
+    padding: 0;
+    margin: 0;
+}
+
+#title {
+    font-size: 100px;
+}
+
+#tagline {
+    font-size: 30px;
+}
+
+.logo-center {
+    opacity: 0.15;
+}
+
+.absolute-center {
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%; 
+    
```

Comment:
For consistency let's remove the newline

Reply:
k